### PR TITLE
[Product excellance] Add scenarios for connecting devices

### PR DESCRIPTION
**Pull request by abeykoon**
## Purpose

Add scenario documentation for "8-Connect-devices-to-enterprise"

### CODE REVIEW

File: product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.1-EI-as-a-MQTT-subscriber/pom.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <version>6.4.0</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>8.1.1-EI-as-a-MQTT-subscriber</artifactId>
+
+    <modules>
+        <!--<module>8.1.1.1-Subscribe-to-MQTT-events-using-MQTT-transport</module>-->
```
Comment (sdkottegoda) [inline product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.1-EI-as-a-MQTT-subscriber/pom.xml]: Commented out module

File: product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.2-EI-as-a-MQTT-events-publisher/pom.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <version>6.4.0</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>8.1.2-EI-as-a-MQTT-events-publisher</artifactId>
+
+    <modules>
+        <!--<module>8.1.2.1-Send-MQTT-message-using-MQTT-transport</module>-->
```
Comment (sdkottegoda) [inline product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.2-EI-as-a-MQTT-events-publisher/pom.xml]: Commented out module

Comment (abeykoon) [reply product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.2-EI-as-a-MQTT-events-publisher/pom.xml]: Removed

File: product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.1-EI-as-a-MQTT-subscriber/pom.xml (hunk 1 above)
Comment (abeykoon) [reply product-scenarios/8-Connect-devices-to-enterprise/8.1-Integration-with-MQTT/8.1.1-EI-as-a-MQTT-subscriber/pom.xml]: Removed

**Pull request merged**